src/structures/GuildMember.ts

```diff
@@ -0,0 +1,17 @@
+import type { APIGuildMember } from 'discord-api-types/v10';
+import type { Client } from '../client';
+import type { Guild } from './Guild';
+import type { User } from './User';
+import { Base } from './Base';
+
+export class GuildMember extends Base {
+	public readonly id = this.user.id;
+
+	public nick = this.raw.nick;
```

Comment:
```suggestion
	public nick?: string = this.raw.nick ?? null;
```

We should be ensuring that there is always *some* value here.